contracts/ClaimsReward.sol

```diff
@@ -207,8 +210,10 @@ contract ClaimsReward is Iupgradable {
      */
     function claimAllPendingReward(uint records) public isMemberAndcheckPause {
         _claimRewardToBeDistributed(records);
-        _claimStakeCommission(records);
-        tf.unlockStakerUnlockableTokens(msg.sender); 
+
+        pooledStaking.withdrawReward(pooledStaking.stakerReward(msg.sender));
+        pooledStaking.unstake(pooledStaking.getMaxUnstakable(msg.sender));
```

Comment:
remove this, so we can keep this function only for claiming rewards (as the name says)

Reply:
ok so basically we change the semantics of this. the button in the UI won't give you all things

Reply:
indeed. the flows are pretty different from a user / UX point of view:
-> withdraw all rewards will be served by this function
-> withdraw tokens you staked for claim assessment, so you can use them in voting again or for something else
-> withdraw stake (unstake) tokens from pooled staking